modules/apps/product-navigation/product-navigation-control-menu-api/src/main/java/com/liferay/product/navigation/control/menu/ProductNavigationControlMenuEntry.java

```diff
@@ -170,6 +170,9 @@ public boolean includeIcon(
 			HttpServletResponse httpServletResponse)
 		throws IOException;
 
+	public boolean isPanelStateOpen(
```

Comment:
Hey @RenanUmV, extract the changes from the `modules/apps/product-navigation/product-navigation-control-menu-api` module to a separate commit.

Thx